res/values/arrays.xml

```diff
@@ -20,25 +20,111 @@
   </string-array>
   <string-array name="pref_theme_entries">
     <item>@string/pref_theme_e_system</item>
-    <item>@string/pref_theme_e_dark</item>
-    <item>@string/pref_theme_e_light</item>
-    <item>@string/pref_theme_e_black</item>
+    <!-- THEMES_ENTRIES_ARRAY_START -->
```

Comment:
Arrays could be outputted to a new file, to make the generator script more maintainable. This is how it's done for layouts, this file is generated: https://github.com/Julow/Unexpected-Keyboard/blob/master/res/values/layouts.xml